### ISSUE TITLE
region enum is not set correctly as shown in cli -help

### ISSUE THREAD

**Issue by skylinezum** [graphcool/graphcool-cli#132]
region option in cli -help asks for us-west-2 or eu- etc... but cli uses `US-WEST-2` instead of `US_WEST_2` enum.

```
graphcool {"data":null,"errors":[{"message":"Variable '$region' expected value of type 'Region' but got: \"US-WEST-2\". Reason: Enum value 'US-WEST-2' is undefined (line 1, column 71):\nmutation addProject($schema: String!, $name: String!, $alias: String, $region: Region) {\n                                                                      ^","locations":[{"line":1,"column":71}]}]} +495ms
```

**Comment by ruurtjan** [comment]
The docs also state `graphcool init --region 'eu-west-1'` (among other places) [here](https://www.graph.cool/docs/reference/cli/commands/creating-and-copying-projects-aetoh3vad6/). I guess the cli should indeed accept dashes. It should internally convert them to one of the graphql enum values.

**Comment by ruurtjan** [comment]
For anyone having the same issue, this is the exhaustive list of the enum values you can use:
```
enum Region {
  EU_WEST_1
  AP_NORTHEAST_1
  US_WEST_2
}
```

**Comment by timsuchanek** [comment]
Fixed in 1.3.13 beta
